server/helpers/events.helper.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+import config from '../../config/config';
+
+export default function sendEvent({ userName, eventType, eventData }) {
+  return axios.post(`${config.eventStreamUrl}`, {
+    clientId: userName,
+    deviceType: 'API',
+    eventTime: new Date().getTime(),
+    eventType,
+    eventData
+  })
+    .then(response => response)
+    .catch(error => error);
```

Comment:
What do you think about just console.loggin the error just so when we are testing locally and don't have an event-stream running (or if it fails on prod we can still use the heroku logs)?